runtime/gc_glue_java/RealtimeGCDelegate.cpp

```diff
@@ -108,7 +108,7 @@ MM_RealtimeGCDelegate::allocateAndInitializeReferenceObjectLists(MM_EnvironmentB
 bool
 MM_RealtimeGCDelegate::allocateAndInitializeUnfinalizedObjectLists(MM_EnvironmentBase *env)
 {
-	const UDATA listCount = MM_HeapRegionDescriptorRealtime::getUnfinalizedObjectListCount(env);
+	const UDATA listCount = MM_RealtimeGCDelegate::getUnfinalizedObjectListCount(env);
```

Comment:
There is no need to specify `MM_RealtimeGCDelegate::` here. This is a member of this class